lib/Basics/debugging.h

```diff
@@ -223,14 +223,15 @@ enable_if_t<is_container<T>::value, std::ostream&> operator<<(std::ostream& o, T
 
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
 
-#define TRI_ASSERT(expr)                                                               \
+#define TRI_ASSERT(expr) /*GCOVR_EXCL_LINE*/                                           \
```

Comment:
This seems to not have effect on my local Clang variant, 🤞 that Jenkins likes it more. Thats the only thing to count.
Even if it does not help, this comment does not cost us anything.

Reply:
FYI: i have also tested to place this EXCL_LINE on all lines of the macro, unfortunatly also no effect on my local variant.

Reply:
Ok, let's see if our Jenkins likes this or not.